bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -280,19 +280,19 @@ public void visit(IssuesIssue issue) {
 
     @Override
     public void visit(ExecutableIssue issue) {
-        addMessage(issue.check(), String.format("Executable files are not allowed (file: %s)", issue.path()), issue.severity());
+        addMessage(issue.check(), String.format("Patch contains an executable file (file: %s)", issue.path()), issue.severity());
```

Comment:
I think for the warning variant, leave out the "file:" part since the sentence is already ending with "file". It looks a bit weird repeating the word.